### ISSUE TITLE
getEdgeProps params not populated

### ISSUE THREAD

**Issue by tj-kev** [flareact/flareact#230]
### Describe the problem:

``` params ``` is not populated in ```getEdgeProps({ params})``` if a slug file is not used. The following works (with [slug].js):
```
/test/[myparam]/anotherdir/[slug]
```

Result:
```
{
  myparam: 'my-param-here',
  slug: 'my-slug-here'
}
```

The following doesn't work (with index.js):
```
/test/[myparam]/anotherdir/
```

Result:
```
{}
```
The value of [myparam] should be provided.

Original discussion at: #229

**Comment by eByte23** [comment]
This should be resolved be PR #232 will need review and merging

**Comment by tj-kev** [comment]
Merged into canary PR #232
